src/types/5.3.x.json

```diff
@@ -0,0 +1,1221 @@
+{
```

Comment:
Should we wait for this one to get merged + released?

https://github.com/PolymeshAssociation/Polymesh/pull/1410

We should wait for it to get merged in case any changes happen during review.

Reply:
This is now merged